_layouts/person.html

```diff
@@ -156,7 +156,9 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
       </div>
 
       <h3 class="person-show-title">
+       <a href="{{ record.item.url }}">
```

Comment:
Note, when changing css you don't need to run the "build" step, only the "serve" step. So should be super duper quick